components/features/SimpleStepsCreate.vue

```diff
@@ -0,0 +1,100 @@
+<template>
+  <section class="container py-5 mt-3">
+    <h2 class="section__heading text-center">
+      {{ heading }}
+    </h2>
+
+    <div class="py-5 row">
+      <div v-for="(step,idx) in steps" :key="step?.title" :class="columnClass">
```

Comment:
I think the column class doesn't need to be done through javascript. Css is capable enough of handling such things.
cc: @partha6686  @dhaladitya108 

`:class="col-xs-12 col-sm-12 col"`

The `col` class will auto accommodate all the steps equally on larger screen.

Reply:
Okay, let me check this one.

Reply:
`class="col-xs-12 col-sm-12 col"`  
Actually, the card is taking the full width. I guess for this to work we have to give some specific width to the cards.

Reply:
yes, this is done so that we can manually specify the no. of columns in a particular row.